### DISCUSSION TITLE
[Drag and Drop]:  Marker (horizontal line) where the element will be inserted

### DISCUSSION THREAD

**Discussion by ElenaOleksenko** [epam/UUI#2342] [Q&A] [answered]
How to insert a marker where the element will be inserted? A horizontal or vertical line.
Thanks for help in advance)

**Comment by AlekseyManetov** [comment, marked as answer]
You should add  `<DropMarker { ...params } />` component to the `DragActor` render for this.
See this example for more details - https://uui.epam.com/documents?id=dragAndDrop&mode=doc&isSkin=null&category=advanced
